holidays/countries/saint_kitts_and_nevis.py

```diff
@@ -0,0 +1,188 @@
+
+    If Sovereign’s Birthday, New Year's Day, Independence Day, or National Heroes Day
+    fall on a Sunday the next following Monday shall be a public holiday.
+
+    Boxing Day—that is the day after Christmas Day, but if Christmas Day falls
+    on a Saturday, then the next following Monday shall be a public holiday, and if
+    Christmas Day falls on a Sunday, then the next following Monday and Tuesday
+    shall be public holidays.
+
+    While Culturama Day (first started in 1974) and Carnival Day are never officially
+    included in the main Chapter 23.23 document, they're de facto added since at least
+    2015 and should be considered as such.
+    """
+
+    country = "KN"
+    supported_categories = (HALF_DAY, PUBLIC, WORKDAY)
+    # %s (observed).
+    observed_label = "%s (observed)"
+    weekend = {SUN}
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, SaintKittsAndNevisStaticHolidays)
+        kwargs.setdefault("observed_rule", SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # Public Holidays Act, Act 7 of 1983 Amendment.
+        if self._year <= 1982:
+            return None
+
+        # Carnival Day.
+        self._add_observed(self._add_new_years_day("Carnival Day"), rule=SUN_TO_NEXT_TUE)
+
+        # Carnival Day - Last Lap.
+        self._add_observed(self._add_new_years_day_two("Carnival Day - Last Lap"))
+
+        # Good Friday.
+        self._add_good_friday("Good Friday")
+
+        # Easter Monday.
+        self._add_easter_monday("Easter Monday")
+
+        # Labour Day.
+        self._add_holiday_1st_mon_of_may("Labour Day")
+
+        # While Sovereign's Birthday is officially listed in the Public Holidays Act,
+        # this was de facto never included in any released calendar since at least 2015.
+
+        # Whit Monday.
+        self._add_whit_monday("Whit Monday")
+
+        first_monday_of_august_holiday_name = (
+            # Emancipation Day.
+            "Emancipation Day"
+            if self._year >= 1998
+            # First Monday of August.
+            else "First Monday of August"
+        )
+        self._add_holiday_1st_mon_of_aug(first_monday_of_august_holiday_name)
+
+        # Culturama Day - Last Lap.
+        self._add_holiday_1_day_past_1st_mon_of_aug("Culturama Day - Last Lap")
+
+        if self._year >= 1998:
+            # National Heroes Day.
+            self._add_observed(self._add_holiday_sep_16("National Heroes Day"))
+
+        # Independence Day.
+        self._add_observed(self._add_holiday_sep_19("Independence Day"))
+
+        # Christmas Day.
+        self._add_observed(self._add_christmas_day("Christmas Day"), rule=SUN_TO_NEXT_TUE)
+
+        # Boxing Day.
+        self._add_observed(self._add_christmas_day_two("Boxing Day"))
```

Comment:
I understand that statement about Christmas Day on Saturday and Sunday as postponing of Christmas Day exactly, not Boxing Day (as it's shown on [Timeanddate](https://www.timeanddate.com/holidays/saint-kitts-and-nevis/christmas-day)). It's also required to import `SAT_SUN_TO_NEXT_MON_TUE`.
```suggestion
        # Christmas Day.
        self._add_observed(self._add_christmas_day("Christmas Day"), rule=SAT_SUN_TO_NEXT_MON_TUE)

        # Boxing Day.
        self._add_christmas_day_two("Boxing Day")
```

Reply:
I double-checked, and it turns out "Dec 27, [2021](https://www.facebook.com/photo/?fbid=3623684614351340)" is listed as `Boxing Day (observed)` as currently implemented instead of `Christmas Day (observed)` as you suggested above.